File: Plugin/src/main/java/dev/rosewood/roseskyblock/world/IslandWorldGroup.java
package dev.rosewood.roseskyblock.world;

import org.bukkit.GameMode;

import java.util.List;

public record IslandWorldGroup(
        String name,
        String displayName,
        IslandWorld startingWorld,
        GameMode gamemode,
        List<IslandWorld> worlds
) {

    // TODO: Add some thingys here.
}

File: Plugin/src/main/kotlin/dev/rosewood/roseskyblock/world/IslandWorldGroup.kt
package dev.rosewood.roseskyblock.world

import org.bukkit.GameMode

data class IslandWorldGroup(
    val name: String,
    val displayName: String,
    val startingWorld: IslandWorld,
    val gamemode: GameMode,
    val worlds: List<IslandWorld>
) {
    // TODO: Add some thingys here
}
